<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LCDGroup</class>
 <widget class="QWidget" name="LCDGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>148</width>
    <height>533</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QPushButton" name="st_transfer">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#st_transfer {
    font: 10pt;
    background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0.05 					#77d42a, stop:1 #5cb811);
    border-style:outset;
    border-width:1px;
    border-radius:8px;
    border-color:#636363;
    padding:8px
}

QPushButton#st_transfer:hover{
    background:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0.05 					#5cb811, stop:1 #77d42a);
}

QPushButton#st_transfer:pressed {
     background: #adff2f;
     border-style: inset;}</string>
     </property>
     <property name="text">
      <string>S-T</string>
     </property>
     <property name="autoRepeatDelay">
      <number>300</number>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_press">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>压力</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="press_chk1">
        <property name="text">
         <string>通道1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="press_lcd1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="press_chk2">
        <property name="text">
         <string>通道2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="press_lcd2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="press_chk3">
        <property name="text">
         <string>通道3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="press_lcd3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="press_chk4">
        <property name="text">
         <string>通道4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLCDNumber" name="press_lcd4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="press_chk5">
        <property name="text">
         <string>通道5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLCDNumber" name="press_lcd5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="press_chk6">
        <property name="text">
         <string>通道6</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLCDNumber" name="press_lcd6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_digital">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>数字量/计时</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="digital_chk1">
        <property name="text">
         <string>通道1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="digital_lcd1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="digital_chk2">
        <property name="text">
         <string>通道2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="digital_lcd2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="digital_chk3">
        <property name="text">
         <string>通道3</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="digital_lcd3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="digital_chk4">
        <property name="text">
         <string>通道4</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLCDNumber" name="digital_lcd4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="group_speed">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="speed_chk">
        <property name="text">
         <string>速度</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLCDNumber" name="speed_lcd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="acceleration_chk">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>加速度</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="acceleration_lcd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="shift_chk">
        <property name="text">
         <string>位移</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="shift_lcd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="smallDecimalPoint">
         <bool>false</bool>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
